err-travel/travel.py

```diff
@@ -0,0 +1,212 @@
+            searched[nick] = [dep, dest, time, date]
+
+        out = unicode(out.strip(codecs.BOM_UTF8), 'utf-8')
+        return out.encode('utf-8')
+
+    def searched_incrementer(self, nick):
+        """Adds 1 minute to previous search departure"""
+        dep, dest, time, date = searched[nick]
+
+        dateobj = datetime.datetime.strptime('{0}-{1}'.format(time,
+                                                              date),
+                                             '%H:%M-%d.%m.%Y')
+        dateobj += datetime.timedelta(seconds=60)
+        date = dateobj.strftime('%d.%m.%Y')
+        time = dateobj.strftime('%H:%M')
+
+        searched[nick] = [dep, dest, time, date]
+        return searched[nick]
+
+    def rootify(self, word):
+        """Return probable root of the word."""
+        if len(word) <= 5:
+            return word
+
+        w = word[::-1]
+        vowels = ['a', 'e', 'i', 'o', 'u', 'y']
+        for x in range(len(word)):
+            if w[x] in vowels and x != 0:
+                return word[:-(x + 1)]
+
+    @botcmd
+    def mhd(self, msg, args):
+        """Get the next BA MHD from A to B by running !mhd A B"""
+        nick = msg.frm.nick
+
+        if '-' in args:
+            args = split_args_by(args, '-')
+        else:
+            args = args.split(" ")
+
+        if len(args) >= 1 and args[0] == 'next':
+            if nick not in searched:
+                return 'No next line'
+            args = self.searched_incrementer(nick)
+
+        if len(args) < 2:
+            return 'Not enough arguments specified. See !help mhd for usage'
+
+        f = args[0]
+        t = args[1]
+
+        if f == t:
+            return 'Not in this universe.'
+
+        time = ''
+        date = ''
+        if len(args) >= 3:
+            time = args[2]
+        if len(args) >= 4:
+            date = args[3]
+
+        r = imhdsk.routes(f, t, time=time, date=date)
+        if len(r) == 0:
+            return
+
+        return self.send_output(nick, f, t, date, result=r)
+
+    @re_botcmd(pattern=TRAVEL_REGEX, prefixed=False)
+    def line_match(self, msg, match):
+        """Search for mhd in BA or bus/train lines in Slovakia
+
+        Examples:
+            bus z mlyny na hlst
+            vlak z BA do TO
+            bus z BA do LM 18:00 20.12.2014
+            spoj zo Zochova no mlyny
+        """
+        f = match.group(1)
+        t = match.group(2)
+        time = match.group(3)
+        date = match.group(4)
+
+        if f == t:
+            return 'Not in this universe.'
+
+        time = '' if time is None else time
+        date = '' if date is None else date
+
+        body = msg.body
+
+        if ('zajtra' in body or 'pozajtra' in body) and date is not '':
+            if 'pozajtra' in body:
+                delta = 2
+            elif 'zajtra' in body:
+                delta = 1
+            date = (datetime.date.today() + datetime.timedelta(days=delta)) \
+                .strftime("%d.%m.%Y")
```

Comment:
You should be consistent in using single or double quoted throughout the file. Please change these into apostrophes (single quotes).